lms/djangoapps/user_tours/toggles.py

```diff
@@ -0,0 +1,15 @@
+"""
+Toggles for the User Tours Experience.
+"""
+
+from edx_toggles.toggles import WaffleFlag
+
+# .. toggle_name: user_tours.tours_disabled
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: This flag disables user tours in LMS.
+# .. toggle_warnings: None
+# .. toggle_use_cases: temporary
```

Comment:
Should we change this to some other value, like `opt_out`?

Reply:
@navinkarkera, good point! I forgot about `opt_out`. This is changed now.